### PR TITLE
Tag docs build after release

### PR DESCRIPTION
**Pull request by saskliutas**
Closes https://github.com/iTwin/presentation/issues/158

### CODE REVIEW

**Comment by ben-polinsky** [comment]
Looks good, thanks!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,6 +35,6 @@ jobs:
         run: |
           git config --local user.email imodeljs-admin@users.noreply.github.com
           git config --local user.name imodeljs-admin
-          pnpm publish-packages -y --branch ${{ github.ref_name }} --message "Version bump [skip actions]"
+          pnpm publish-packages -y --branch ${{ github.ref_name }} --message "Version bump [skip actions][Publish Docs]"
```
Comment (grigasp) [inline .github/workflows/release.yml:38]: just for consistency I'd prefer `[publish docs]` instead of `[Publish Docs]

**Comment by grigasp** [comment]
In case we need to patch an older release, e.g. let's say we're on `4.1` and want to release `4.0.1`, it looks like we're going to append `[publish docs]` in both cases, which will cause the tag to be added. Should we also check if the release is made from `master`?

**Comment by saskliutas** [comment]
> In case we need to patch an older release, e.g. let's say we're on `4.1` and want to release `4.0.1`, it looks like we're going to append `[publish docs]` in both cases, which will cause the tag to be added. Should we also check if the release is made from `master`?

Docs pipeline should be triggered only by push to master.

File: .github/workflows/release.yml (hunk 1 above)
Comment (saskliutas) [reply .github/workflows/release.yml]: Fixed

**Pull request merged**